basis-library/primitive/prim-int-inf.sml

```diff
@@ -29,8 +29,10 @@ structure IntInf =
       val notb = _prim "IntInf_notb": int * C_Size.t -> int;
       val orb = _prim "IntInf_orb": int * int * C_Size.t -> int;
       val quot = _prim "IntInf_quot": int * int * C_Size.t -> int;
+      val quotRem =
+         _prim "IntInf_quotRem": int * int * C_Size.t * C_Size.t -> int vector;
```

Comment:
See discussion below, but this will need to be revised to either: `int * int * C_Size.t * C_Size.t * C_Size.t -> int vector` or to `int * int * C_Size.t -> int vector`.